### DISCUSSION TITLE
获取公众号id时总是提示失败

### DISCUSSION THREAD

**Discussion by ai668chen** [qiye45/wechatDownload#236] [Q&A] [answered]
第一次运行时可以获取到id，由于时间期限没有选对，导致下载公众号下全部内容，没有找到停止运行按钮，所以强制关闭了程序，再次打开使用后，就一直无法获取公众号id，如何解决？

**Comment by qiye45** [comment, marked as answer]
试试新版本，看看[issue](https://github.com/qiye45/wechatDownload/issues/9)
